src/components/PostEditor/PostEditor.js

```diff
@@ -27,6 +27,7 @@ import { PROJECT_CONTRIBUTIONS } from 'config/featureFlags'
 import { MAX_POST_TOPICS } from 'util/constants'
 import { sanitizeURL } from 'util/url'
 import Tooltip from 'components/Tooltip'
+import setQuerystringParam from 'store/selectors/setQuerystringParam'
```

Comment:
this is not a selector. selectors are just about getting data. we already have a function util/navigation.js#addQuerystringToPath which does almost this. you can tweak it to support no path, or add another function to util/navigation.js

Reply:
Doh! Thanks for pointing me to the right category :)

Reply:
Okay, moved the function to util/navigation.js as addQuerystringToPath would have been kinda mangled by this feature.